### ISSUE TITLE
BUG: Images in figure environments cannot be scaled

### ISSUE THREAD

**Issue by ratnanil** [ResearchMethods-ZHAW/Statistik#12]
```
:::{layout-nrow="1"}

![](./myMediaFolder/media/image90.png)

![](./myMediaFolder/media/image91.png)

:::
```

as soon as I wrap this like so, I cannot resize the images to 80%

**Comment by DLND8** [comment]
I think this is currently not possible. Or did you find a solution for that @ratnanil?

**Comment by ratnanil** [comment]
maybe related to https://github.com/quarto-dev/quarto-cli/issues/7074

**Comment by ratnanil** [comment]
this bug is solved in version `1.4.489`. @DLND8 : can you update your quarto version to the newest pre-release?
